main.nf

```diff
@@ -481,17 +481,10 @@ process wgdParalogs {
     cd $PWD
     echo "NF internal work directory for [wgdParalogs] process:\n\$processDir\n" >> $logs_folder/wgd_paralogs.log
 
-    if [ -z \${NSLOTS+x} ]; then
-        nThreads=$params.nThreadsParalogs
```

Comment:
Did you also remove the parameter `nThreadsParalogs`? Or do we still need it for something else?